model/common_structures.smithy

```diff
@@ -59,3 +59,34 @@ structure Hits{
     _source: Document,
     fields: Document
 }
+
+
+structure PitSegments {
```

Comment:
I think we can move `PitSegments` to `cat/pit_segments/structures.smithy` since its not a common structure used across various API models.

Reply:
Yeah, i got it only yesterday at meeting.